extensions/typescript/src/package.test.ts

```diff
@@ -7,7 +7,9 @@ describe('resolvePackageRepo', () => {
     it('resolves string repo', async () => {
         const api = new API()
         const mock = sinon.stub(api, 'resolveRepo')
-        mock.callsFake(repo => Promise.resolve(repo))
+        mock.callsFake(repo =>
+            Promise.resolve({ name: repo, isFork: false, isArchived: false })
```

Comment:
Do we also need to cover the case where `isFork` is not defined on the object?

Reply:
It's always defined (the API.resolveRepo will return false for isFork/isArchived if introspection shows the fields aren't defined on older instances).